src/components/Radio/Radio.stories.js

```diff
@@ -28,6 +28,8 @@ export const Default = (args) => ({
       :label="label"
       :disabled="disabled"
       :required="required"
+      v-model="internalValue"
```

Comment:
`InternalValue` declaration is missing in the **Default** storie, there should be (Like storie **Selected**):
```js
data: () => ({
    internalValue: 'Example',
  }),
```

Reply:
good observation, thanks!